src/Actions/Action.php

```diff
@@ -52,4 +54,9 @@ public function setSelectedTargets($targets)
         $this->selectedTargets = $targets;
         return $this;
     }
+
+    public function getActionConfirmationMessage()
+    {
+        return Translation::useTranslationPrefix(Str::camel($this->actionConfirmationMessage), $this->actionConfirmationMessage ? $this->actionConfirmationMessage.'?' : $this->actionConfirmationMessage);
```

Comment:
aixo ho veig massa complicat perque estigui aquí, no ho pot fer tot el mateix translation? i fer nomes
Translation::translate($this->actionConfirmationMessage)